client/packages/programs/src/hooks/usePatientStore.ts

```diff
@@ -1,13 +1,19 @@
 import { zustand } from '@openmsupply-client/common';
+import { PatientRowFragment } from '../../../system/src/Patient/api';
 
 // stores the `documentName` for the currently viewed patient document
 interface PatientStoreState {
   documentName: string | undefined;
   setDocumentName: (documentName: string | undefined) => void;
+  currentPatient: PatientRowFragment | undefined;
+  setCurrentPatient: (patient: PatientRowFragment) => void;
 }
 
 export const usePatientStore = zustand<PatientStoreState>(set => ({
   documentName: undefined,
   setDocumentName: (documentName: string | undefined) =>
     set(state => ({ ...state, documentName })),
+  currentPatient: undefined,
+  setCurrentPatient: (patient: PatientRowFragment) =>
```

Comment:
Any reason why not allow to unset it? i.e. `patient: PatientRowFragment | undefined`

Maybe good to clean up when leaving a patient but not sure how easy it is to reliable unset it...

Reply:
Okay, have added `undefined` as an allowed input as suggested: ddd8cb495

However, I haven't added any code to "unset" the patient, as I think the logic to figure out the correct time to do so would be unnecessarily complicated (and we don't "unset" the documentName at all here either).

Reply:
cool thanks, yes agree was more a question / nit